website/content/labs/lab1.md

```diff
@@ -2,26 +2,39 @@
 
-You will each have a personal Git repository that is hosted on a central server run by the CS department (the software running on that server is [GitLab](https://about.gitlab.com/), so we will often refer to it as "the GitLab server"). You may have also heard of [GitHub](https://github.com/), a web-based hosting service for Git repositories. We will be using GitHub later in the quarter for the course project, but will be using GitLab for the labs, since it will be easier for us to access your work there (since it is tied to CNetID authentication, etc.). We will also be browsing some Git repositories hosted on GitHub, but you will not need a GitHub account just yet.
+For each lab assignment, a Git repository will be created for you on [GitHub](https://github.com/), a web-based hosting service for Git repositories. However, before that repository can be created for you, you need to have a GitHub account. If you do not yet have one, you can get an account here: https://github.com/join. Once you create your account, you may want to get the [Student Developer Pack](https://education.github.com/pack), which will give you access to a lot of other features (please note that having the Student Developer Pack is not necessary for CS 220; it's just a nice benefit you get as a student) 
+
+## Creating your lab repository
+
+For each lab assignment, we will provide you with an *invitation URL* that will allow you sign up for the lab assignment on GitHub, and which will result in the creation of a repository called `2020-labN-GITHUB_USERNAME` inside our `cmsc22000-labs` organization on GitHub (a GitHub "organization" is basically a way to group together related repositories). For example, if your GitHub username is `jrandom`, your repository will be called `2020-lab1-jrandom`. This repository will be private, and can only be viewed by you and the CS 220 course staff.
+
+## Where should you do this lab?
+
+For this lab, we will specifically need you to work on a CS machine, which means you will have to connect remotely to CS environment. You can do this in one of two ways:
+
+- Using SSH, which will allow you to open a command-line terminal on a CS machine. The CS techstaff provides [detailed instructions](https://howto.cs.uchicago.edu/remote_access) on how to do this.
+- Connecting to a [Virtual Desktop](https://howto.cs.uchicago.edu/techstaff:vdesk). This will show you, on your computer, effectively the exact same desktop you would see if you physically logged into a CSIL machine. However, this option may not work well for low-bandwidth Internet connections. 
+
+In future labs, there will be more flexibility on where you can do your work, including working locally on your own computer.
```

Comment:
is this virtualbox? are the instructions I wrote up on the techstaff wiki still good?

Reply:
Not virtualbox- techstaff introduced a website this quarter that allows you to connect to a graphical interface through a web browser (it's actually pretty cool, but I still prefer ssh/vbox).

The instructions for setting up vm's on the techstaff wiki are still great; I'm just assuming Borja didn't wanna recommend setting up the VM and then throwing the students down a rabbit hole. However, I was wondering why the Vm wasn't recommended here?

Reply:
Queue in "my virtual machine doesn't work" messages on Slack

Reply:
We're going to point them to the VM in the developer's guide. For this first lab, the lack of VM mentions is intentional; I think it's important they do at least one exercise in a CS environment so they're clear on how to access it remotely (there will likely be a fair number of students who have never used SSH because they were never placed in a position to do so)